### ISSUE TITLE
Handle error when Netlify changes chunk files while user is viewing website

### ISSUE THREAD

**Issue by Turbotailz** [lucko/LuckPermsWeb#261]
## Scenario

- make change to `production` branch
- Netlify receives update and builds new files
- meanwhile, user is viewing the website
- user has started viewing the site BEFORE Netlify builds new files
- each page has its own chunk file which is requested whenever the user navigates to the page
- user tries to navigate to a page AFTER Netlify builds files
- browser requests an old chunk file
- nothing happens
- user: ![](https://media1.giphy.com/media/lkdH8FmImcGoylv3t3/giphy.gif)

Basically just need to figure out the best way to handle this. I think refreshing the page if the chunk file is not found would be the best option.
